_projects/SP19-<CostcoChaos>.md

```diff
@@ -0,0 +1,14 @@
+---
+layout: project
+title: <CostcoChaos>
+
+desc: |
+ <CostcoChaos Final Project for CS498 Virtual Reality, Spring 2019>
+
+dl: <https://drive.google.com/drive/folders/1Krn0JwMWHsDliuNhRGrcab7Ektc9ZGUp?usp=sharing>
+semester: Spring 2019
+team: <29>
+youtube: <vGJOpRMu11U>
+featured: false
+---
+<full video link: https://www.youtube.com/watch?v=vGJOpRMu11U&feature=youtu.be>
```

Comment:
The YouTube link will be posted in the page, for an example of what it might look like see [here](https://cs498vr.cs.illinois.edu/projects/SP19-FactorySimulator3000.html), so you don't need this line.
```suggestion
```